subsys/bluetooth/host/gatt.c

```diff
@@ -1634,18 +1638,37 @@ int bt_gatt_indicate(struct bt_conn *conn,
 		     struct bt_gatt_indicate_params *params)
 {
 	struct notify_data data;
+	const struct bt_gatt_attr *attr;
 	u16_t handle;
 
 	__ASSERT(params, "invalid parameters\n");
 	__ASSERT(params->attr, "invalid parameters\n");
 
-	handle = params->attr->handle ? : find_static_attr(params->attr);
+	attr = params->attr;
+
+	handle = attr->handle ? : find_static_attr(attr);
 	if (!handle) {
```

Comment:
It looks to me like the user still has to provide a valid `attr`? Or am I misunderstanding?

Reply:
Yes, it is used as start location so it can either point to the start of its services, more likely, or handle 0x0001.